### ISSUE TITLE
Search field needs a screen-readable label

### ISSUE THREAD

**Issue by Norric1Admin** [ONSdigital/sdg-indicators#3092]
The indicator Search facility form field contains label text that is hidden by CSS which means screen reader users may not be able to determine the form elements function or purpose. This is consistent throughout the website. Ensure that form elements contain a label that is visible to all users and descriptive to help them identify the elements function or purpose (`.navbar #search label span {display: none;}`)

**Comment by Norric1Admin** [comment]
![](https://github.trello.services/images/mini-trello-icon.png) [Accessibility tickets](https://trello.com/c/E40VA79G/549-accessibility-tickets)

**Comment by Norric1Admin** [comment]
From DAC:
‘Unlabelled Form Fields’ says that the search feature contains label text hidden by CSS, and also not to use placeholder text. Could you explain this problem please?
"If the CSS class display: none; is removed and the following CSS is used, the ‘Search:’ label will be visible to screen reader users."

**Comment by LucyGwilliamAdmin** [comment]
## Description

## Screenshots
**Before**

**After**


## Successfully tested on...
- [] Windows: IE11
- [] Windows: Chrome
- [ ] Windows: Firefox
- [ ] Windows: Edge
- [ ] macOS: Chrome
- [ ] macOS: Firefox
- [ ] iOS: Safari 9.3
- [ ] iOS: Chrome
- [ ] Samsung: Chrome
- [ ] Samsung: Samsung Internet

## User feedback and problems encountered...

**Comment by Norric1Admin** [comment]
This is not especially clear, so me and @SavvasStephanides will speak to DAC. You can view the original recommendation on the [DAC Accessibility Audit](https://share.sp.ons.statistics.gov.uk/sites/SusIneq/SDG/Sample/NRP/Accessibility/ONS%20SDG%20and%20ArcGIS%20-%20DAC%20Accessibility%20Audit%20-%20240419.pdf) (p.98: "Unlabelled Form Fields").

**Comment by Norric1Admin** [comment]
![](https://github.trello.services/images/mini-trello-icon.png) [Search field needs a screen-readable label](https://trello.com/c/3nmyI6vN/615-search-field-needs-a-screen-readable-label)

**Comment by Norric1Admin** [comment]
DAC advised that this recommendation was a mistake, produced through their automated testing. We will close the Issue and consider it completed, unless DAC revises the recommendation following extra tests.

**Comment by samuelwillis8** [comment]
Tested using NVDA screenreader on the 4 windows browsers. On each, there was some kind of 'indicator search' announcement when tabbing to the search bar

**Comment by Norric1Admin** [comment]
@samuelwillis8 
Great news! In that case the Issue will be closed :+1:

**Comment by Norric1Admin** [comment]
Also DAC: following further tests, placeholder text is not suitable for certain screenreaders. On their recommendation I've switched aria-hidden to `false`, but this has split the search field into two parts. 
![image](https://user-images.githubusercontent.com/37659239/61451736-0357aa00-a952-11e9-83a5-9b42cdd3b558.png)

Testing further to find a solution.

**Comment by Norric1Admin** [comment]
Turns out the search field is super temperamental and can't be edited or overriden without the splitting into two components. @SavvasStephanides any thoughts?

**Comment by LucyGwilliamAdmin** [comment]
It's fine on here https://sdg.mango-solutions.com/search-button/

**Comment by LucyGwilliamAdmin** [comment]
the problem is that you took what I had done for a different feature which turns the search icon into an actual button.

**Comment by Norric1Admin** [comment]
@LucyGwilliamAdmin merged her changes across from open-sdg and this is working on [dac-final](https://sdg.mango-solutions.com/dac-final/).

**Comment by Norric1Admin** [comment]
Added an aria label, this should be the final step: https://github.com/ONSdigital/sdg-indicators/commit/f1ae874fb86ccede8548033594f60e7bb2df1a04

**Comment by LucyGwilliamAdmin** [comment]
This is now in Open SDG V0.8.0 so closing this issue
